SFR-1410_ManifestScript.py

```diff
@@ -0,0 +1,49 @@
+import boto3
+import json
+import logging
+
+from urllib.parse import urlparse, parse_qsl, urlencode, urlunparse
+
+logging.basicConfig(filename='manifest.log', encoding='utf-8', level=logging.INFO)
+
+s3_client = boto3.client("s3")
+
+listObjects = s3_client.list_objects_v2(Bucket='drb-files-qa', Prefix='manifests/muse/')
```

Comment:
By default this will only load the first 1000 objects it finds from s3 (see the `MaxKeys` parameter in the boto3 documentation). So I think we need to add some paging capability here to load everything.

It might make sense to break this up into a few methods: 1 to load a batch and another to update all of the records in the batch.

For the behavior we can use to do this see the `IsTruncated` object in the `list_objects_v2` response documentation.

Reply:
I decided to use a Paginator to iterate through all the objects instead.